unitOneAssessmentPractice.js

```diff
@@ -14,22 +16,43 @@ let assert = require('assert')
 // Uncomment out the next line to test your solution
 // runQ2Tests()
 
+const containsSeven = (num) => {
+  let stringToNum = num.toString()
+  return stringNum.includes('7')
+}
+
 // Question Three:
 
 // Write a function called capitalizeTheAs the capitalizes all of the lowercase "a"s in a string leaving all other characters the same
 
 // Uncomment out the next line to test your solution
 // runQ3Tests()
 
+
+function capitalizeTheAs(str) {
```

Comment:
Be consistent with your function styles - you have some functions written in ES5 and some in ES6. We lean more towards ES6.